src/openforms/variables/registry.py

```diff
@@ -0,0 +1,14 @@
+from openforms.plugins.registry import BaseRegistry
+
+
+class Registry(BaseRegistry):
+    """
+    A registry for static variables.
+    """
+
+    module = "variables"
+
+
+# Sentinel to provide the default registry. You can easily instantiate another
+# :class:`Registry` object to use as dependency injection in tests.
+static_variables_register = Registry()
```

Comment:
I'd rename this to `register_static_variable` so it reads a bit more natural when used as a decorator.

In the `FormVariable.get_static_data` module you can alias the import to `static_variables_register` so that also still reads like proper English :)